photon-core/src/main/java/org/photonvision/vision/pipe/impl/CropPipe.java

```diff
@@ -54,4 +54,14 @@ protected CVMat process(CVMat in) {
     public static boolean fullyCovers(Rect rect, Mat mat) {
         return rect.x <= 0 && rect.y <= 0 && rect.width >= mat.width() && rect.height >= mat.height();
     }
+    @Override
+    public void setParams(Rect params) {
+        if(params.width < 16 || params.height < 16)
+        {
+            params.width = 16;
+            params.height = 16;
+        }
```

Comment:
What happens if params.width == 200 and params.height == 1?

Reply:
Your right I'll fix that

Reply:
(And if your unit tests don't currently catch this, you should add some tests)